frontend/src/qri/queryParams.ts

```diff
@@ -0,0 +1,6 @@
+export interface QueryParams {
+  q?: string
+  sort?: 'name' | 'recentlyupdated'
```

Comment:
both of these should be required. Consider adding the empty string as a possible sort value